e2e-tests/scripts/ci-build.bat

```diff
@@ -1,18 +1,24 @@
-cd %SAMPLES_ASPNET_WEBFORMS_HOME%\e2e-tests
+rmdir /Q /S %SAMPLES_ASPNET_WEBFORMS_HOME%\MYGIT
+git clone git@github.com:okta/samples-aspnet-webforms.git MYGIT
+cd %SAMPLES_ASPNET_WEBFORMS_HOME%\MYGIT
+git checkout %SHA%
+git rev-parse --short HEAD > MY_SHORT_SHA
+
+cd %SAMPLES_ASPNET_WEBFORMS_HOME%\MYGIT\e2e-tests
 call npm install
 call npm run pretest
 
 cd %IIS_EXPRESS_HOME%
-call start iisexpress.exe /path:%SAMPLES_ASPNET_WEBFORMS_HOME%\okta-hosted-login\dist\okta-aspnet-webforms-example
-cd %SAMPLES_ASPNET_WEBFORMS_HOME%\e2e-tests
+call start iisexpress.exe /path:%SAMPLES_ASPNET_WEBFORMS_HOME%\MYGIT\okta-hosted-login\dist\okta-aspnet-webforms-example
+cd %SAMPLES_ASPNET_WEBFORMS_HOME%\MYGIT\e2e-tests
```

Comment:
This is just to follow the naming convention set by the eng-prod team. 
I'm changing this script so that nightly runs and bacon runs (for PRs) use the same directory.